src/main/java/esa/sen2vm/utils/Sen2VMFilesConstants.java

```diff
@@ -0,0 +1,34 @@
+package esa.sen2vm;
+
+public class Sen2VMFilesConstants {
```

Comment:
Do we need all this constants ?

Reply:
Done here: https://github.com/sen2vm/sen2vm-core/commit/6365dea45094dc8eaaa439d211322f625e249e26